app/graphql/types/input/new_organization_type.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+Types::Input::NewOrganizationType = GraphQL::InputObjectType.define do
+  name 'NewOrganization'
+
+  argument :name, !types.ID, 'Name of the new organization'
+  argument :displayName, types.String, nil, as: :display_name
+  argument :description, types.String
+end
```

Comment:
I added three test cases to check if all types, fields and arguments have descriptions.